packages/flutter/lib/src/material/chip.dart

```diff
@@ -1806,15 +1802,12 @@ class _RawChipState extends State<RawChip> with MaterialStateMixin, TickerProvid
       child: _wrapWithTooltip(
         widget.deleteButtonTooltipMessage ?? MaterialLocalizations.of(context).deleteButtonTooltip,
         widget.onDeleted,
-        GestureDetector(
-          key: deleteIconKey,
-          behavior: HitTestBehavior.opaque,
-          onTap: widget.isEnabled
-            ? () {
-                Feedback.forTap(context);
-                widget.onDeleted!();
-              }
-            : null,
+        InkWell(
+          // Radius should be slightly less than the full size of the chip.
```

Comment:
Why "slightly less"? Do we just want to make sure it doesn't paint outside of the chip?

Reply:
Yeah, I could make it half, but then the last part of the splash paints outside the chip.  Aesthetically, I didn't think it looked right.  I'm happy to make it 0.5, though if you think it's better to be agnostic about it.

Reply:
No that's fine. I was just curious about the reasoning behind it. 0.45 looked good to me.